services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,19 +44,46 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  if(xQueueSend(thermalMgrQueueHandle, event, (TickType_t) 10) != pdTRUE) {
```

Comment:
- [ ] check that the queue handle isn't null before use, returning invalid state if it is
- [ ] check that the event isn't null
- [ ] use 0 for the block time as we don't want to block in isr